tree/tree/src/InternalTreeUtils.cxx

```diff
@@ -18,8 +17,6 @@
 #include "TString.h"
 #include "TSystem.h"
 #include "TSystemFile.h"
-#include "TTree.h"
-#include "TVirtualIndex.h"
```

Comment:
Why those `#include` are being removed?

Reply:
Because they are already there in the header file. Probably I should make this a separate commit though

Reply:
It is up for debate but it is usually better to include "all you use" directly rather than rely on transitive inclusion.  However you are right the case of being included by the corresponding header file is an exception to that 'rule'.  So yes, it should be a separate (NFC) commit.